ChangeLog

```diff
@@ -1,3 +1,25 @@
++ openCryptoki 3.24
+- Add support for building Opencryptoki on the IBM AIX platform
+- Add support for the CCA token on non-IBM Z platforms (x86_64, ppc64)
+- Add support for protecting tokens with a token specific user group
+- EP11: Add support for combined CKA_EXTRACTABLE and CKA_IBM_PROTKEY_EXTRACTABLE
+- CCA: Add support for Koblitz curve secp256k1
+- CCA: Add support for IBM Dilithium (CKM_IBM_DILITHIUM)
+- CCA: Add support for RSA-OAEP with SHA224, SHA256, and SHA512 on en-/decrypt
```

Comment:
There is no SHA384?

Reply:
typo, SHA256 was there before, new is SHA224, SHA384 and SHA512.